packages/directlinespeech/.prettierrc.yml

```diff
@@ -0,0 +1,10 @@
+arrowParens: avoid
+bracketSpacing: true
+jsxBracketSameLine: false
+printWidth: 120
+proseWrap: preserve
+quoteProps: as-needed
+semi: true
+singleQuote: true
+tabWidth: 2
+trailingComma: none
```

Comment:
❤

Reply:
Prettier is required for Jest `toMatchInlineSnapshot` function. 😂